README.md

```diff
@@ -1 +1,45 @@
-# Zunair-Mobile-Setup-Website
\ No newline at end of file
+
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+
+- [🔭 Future Features](#future-features)
+ 
+- [👥 Authors](#authors)
+
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Zunair-mobile-setup-website] <a name="about-project"></a>
+
+
+
+**[Zunair-mobile-setup-website]** is a project to demonstrate ability in making and cloning the
+the mobile version of website by adding linters and adding styling the html file using css stylesheet.
+
+
+
+
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+
+👤 **Author**
+
+- GitHub: [@zunairkhan811](https://github.com/zunairkhan811)
+
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **[Will_try_update_html_file]**
+- [ ] **[Will_try_to_update_css_file]**
+- [ ] **[will_make_javascript_file]**
```

Comment:
I have used a partial template because that is enough according to my project requirement so I am leaving it optional.

Reply:
> I have used a partial template because that is enough according to my project requirement so I am leaving it optional.

@zunairkhan811, You need to implement the full version of the `README` in order to pass this project. The only optional section is the FAQ section in the `README`. Kindly use [Microverse Readme template](https://github.com/microverseinc/readme-template) and customize the information as per your project.

You can consider the next reviewers [suggestion.](https://github.com/zunairkhan811/Zunair-Mobile-Setup-Website/pull/1#discussion_r1067408521)